businessRules/users.js

```diff
@@ -0,0 +1,61 @@
+const { v4 } = require('uuid');
+const md5 = require('md5');
+const { createUser } = require('../models/users');
+const { getUsers, setUsers } = require('../utils/storage');
+
+function registerUser(userName, password, firstName, lastName) {
+    const noFirstName = 'Anonymous';
+    const noLastName = 'Incognito';
+
+    if (!firstName && !lastName) {
+        firstName = noFirstName;
+        lastName = noLastName;
+    }
+
+    if (!firstName) {
+        firstName = noFirstName;
+    }
+
+    if (lastName === null) {
+        lastName = noLastName;
+    }
```

Comment:
Can you simplify here? Instead of 3 `if` statements try to use only 2.

Reply:
mmm, to be honest I can't even imagine how, if only  something like this:
if( !firstName) {
} else if ( !lastName) {
}
Something like that?..

Reply:
Or perhaps simply delete this one
    if (!firstName && !lastName) {
        firstName = noFirstName;
        lastName = noLastName;
    }

Reply:
Updated and tested.

Reply:
Well done